service/e2e_tests/test_search.py

```diff
@@ -152,6 +152,19 @@ def setUpClass(cls):
             body=generate_nl_material(educational_levels=["HBO"], technical_type="video", source="edusources")
         )
 
+    def test_pre_filter_search(self):
+        self.selenium.get(self.live_server_url + "?demo=true")
```

Comment:
Perhaps we should wait with this story until we remove the demo flag from the feature? Once we do all kinds of tests are going to break and need to be rewritten. This entire test is probably obsolete after such a rewrite, because we test a lot more in those other tests than what is tested here.